engine/src/cmd/drawable.cpp

```diff
@@ -62,17 +62,17 @@ inline static float perspectiveFactor(float d) {
     }
 }
 
-Drawable::Drawable() :
-        halos(new HaloSystem()),
-        animatedMesh(true),
-        activeAnimation(0),
-        timeperframe(3.0),
-        done(true),
-        activeMesh(0),
-        nextactiveMesh(1),
-        infiniteLoop(true),
-        loopCount(0),
-        curtime(0.0) {
+Drawable::Drawable() : halos(new HaloSystem()),
```

Comment:
This would have been easier (trivial) to review without the whitespace change.

Reply:
It also would have been formatted incorrectly. 😉

Reply:
> This would have been easier (trivial) to review without the whitespace change.

you can disable whitespace changes in the PR by changing the PR settings - or adding IIRC `&wh=1` to the URL.